packages/dcos-metrics/build

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+export GOPATH=$GOPATH:/pkg
+mkdir -p /pkg/src/github.com/dcos
+# Create the GOPATH for the go tool to work properly
+mv /pkg/src/dcos-metrics /pkg/src/github.com/dcos/dcos-metrics
+cd /pkg/src/github.com/dcos/dcos-metrics
+
+make build
+
+mkdir -p $PKG_PATH/bin
+mv /pkg/src/github.com/dcos/dcos-metrics/build/collector/dcos-metrics-collector-* $PKG_PATH/bin/dcos-metrics
+
+# We build this for integration tests
+mv /pkg/src/github.com/dcos/dcos-metrics/build/statsd-emitter/dcos-metrics-statsd-emitter-* $PKG_PATH/bin/statsd-emitter
+
+# Create the service file for all roles 
+agent_service="$PKG_PATH/dcos.target.wants_slave/dcos-metrics.service"
```

Comment:
Two systemd services with different contents should never have the same name when we're shipping them. renaming them all to dcos-metrics.service will make debugging difficult on hosts (people assume same name -> same contents).